daemon/daemon_unix.go

```diff
@@ -1151,3 +1154,20 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) setupSeccompProfile() error {
+	b, err := ioutil.ReadFile(seccompProfilePath)
+	if err != nil {
+		// TODO(runcom): generate a default seccomp profile if system wide isn't present?
```

Comment:
This is also causing CI to fail because on integration tests start the daemon doesn't come up (not `/etc/docker/seccomp.json`)